website/sidebars.json

```diff
@@ -1278,14 +1278,23 @@
 				},
 				"app-development/widgets/form-widgets/toggle",
 				"app-development/widgets/form-widgets/switch",
-				"app-development/widgets/form-widgets/date-time-datetime",
+				{
+					"type": "category",
+					"label": "Date, Time and Datetime",
+					"items": [
+						"app-development/widgets/form-widgets/date-time-datetime",
+						"how-tos/timezone-date-datetime-time"
```

Comment:
Two new files are added, but the **setTimezone** document is not included in the sidebar. Please fix it.